app-frontend/src/app/components/pages/project/analyses/visualize/index.html

```diff
@@ -1 +1,24 @@
-visualize
+<div class="page-content-container">
+    <div class="page-header wide">
+        <h3>Key metrics</h3>
+    </div>
+    <div class="page-card analysis-maps-card">
+      <div class="row">
+        <div class="column-4 flex-display analysis-viz-container" ng-repeat="analysis in $ctrl.analyses track by $index">
+          <rf-analysis-map-item
+              class="panel panel-off-white project-item"
+              analysis-id="analysis.id"
+              analysis-tile="analysis.analysisTile">
+              <item-selector>
+                <rf-list-item-selector
```

Comment:
I'm not sure I understand why we can select these and what we could do with those we select.